test/Feature/VarArgAlignment.c

```diff
@@ -0,0 +1,88 @@
+/* This test is for the alignment of variadic arguments.  In
+   particular, on x86 arguments > 8 bytes (long double arguments in
+   test1) should be 16-byte aligned, unless they are passed byval
+   (struct mix arguments in test2) */
+
+// RUN: %clang %s -emit-llvm %O0opt -c -g -o %t1.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --exit-on-error --output-dir=%t.klee-out %t1.bc | FileCheck %s
+
+#include <stdarg.h>
+#include <assert.h>
+#include <stdio.h>
+
+void test1(int x, ...) {
+  va_list ap;
+  va_start(ap, x);
+  int i1 = va_arg(ap, int);
```

Comment:
Wouldn't it be nice to test `char` and `short` too?

Reply:
You are right, although in this case I'd be hesitant to change it, as I played with these tests to check various alignments.   But we could add another test if you think it's necessary.

Reply:
Actually, I was just reminded by the compiler that you cannot pass chars and shorts to a variadic function:
```
warning: second argument to 'va_arg' is of promotable type
      'short'; this va_arg has undefined behavior because arguments will be
      promoted to 'int' [-Wvarargs]
```